File: online-judge-solutions/Codeforces/1301A.cpp
//https://codeforces.com/problemset/problem/1301/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int t;
	cin>>t;
	while(t--){
		string a,b,c;
		cin>>a>>b>>c;
		int r=0;
		for(int i=0;i<a.size();i++){
			if(c[i]==b[i] || c[i]==a[i]) r++;
		}
		r==a.size()? cout<<"YES\n":cout<<"NO\n";
	}
	return 0;
}

File: online-judge-solutions/Codeforces/1301A.py
#https://codeforces.com/problemset/problem/1301/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
t=int(input())
for x in range(0,t):
    a,b,c=input(),input(),input()
    r=0
    for i in range(len(a)):
        if(c[i]==b[i] or c[i]==a[i]): r+=1
    print("YES") if r==len(a) else print("NO")
